spec/services/idv/aamva_state_maintenance_window_spec.rb

```diff
@@ -65,4 +65,25 @@
       end
     end
   end
+
+  describe 'MAINTENANCE_WINDOWS' do
+    described_class::MAINTENANCE_WINDOWS.each do |state, windows|
+      it 'has a string for a key and an array as a value' do
+        expect(state).to be_a(String)
+        expect(windows).to be_an(Array)
+      end
+
+      windows.each do |window|
+        it "consists of a valid cron expression and numeric duration (#{state})" do
```

Comment:
While it feels weird, I interpolate state name here so the test is a little easier to follow. The output looks like:

```
    consists of a valid cron expression and numeric duration (IN)
    consists of a valid cron expression and numeric duration (TN) (FAILED - 1)
    consists of a valid cron expression and numeric duration (MD)
    consists of a valid cron expression and numeric duration (VA)
    consists of a valid cron expression and numeric duration (DC)
    consists of a valid cron expression and numeric duration (MA)
```

Makes it a lot easier to investigate a failure.